api/tests/prescription/campaign/integration/infrastructure/repositories/campaign-profiles-collection-participation-summary-repository_test.js

```diff
@@ -257,6 +260,269 @@ describe('Integration | Repository | Campaign Profiles Collection Participation
+          });
+
+          await databaseBuilder.commit();
+
+          // when
+          const results = await campaignProfilesCollectionParticipationSummaryRepository.findPaginatedByCampaignId(
+            campaign.id,
+          );
+
+          // then
+          expect(results.data[0]).to.deep.include({
+            pixScore: recentParticipationPixScore,
+            previousPixScore: null,
+            previousSharedAt: null,
+            evolution: null,
+          });
+        });
+
+        it('should return null for previous participations infos and evolution when participations are from different learners', async function () {
+          // given
+          const otherUserId = databaseBuilder.factory.buildUser({}).id;
+
+          const otherOrganizationLearnerId =
+            databaseBuilder.factory.prescription.organizationLearners.buildOrganizationLearner({
+              firstName: 'Martin',
+              lastName: 'Sapin',
+              organizationId,
+              otherUserId,
+            }).id;
+
+          const otherOrganizationLearnerParticipation = databaseBuilder.factory.buildCampaignParticipation({
+            campaignId: campaign.id,
+            userId: otherUserId,
+            organizationLearnerId: otherOrganizationLearnerId,
+            pixScore: 0,
+            isImproved: true,
+            createdAt: new Date('2024-01-01'),
+            sharedAt: new Date('2024-01-02'),
+            status: CampaignParticipationStatuses.SHARED,
+          });
+
+          await databaseBuilder.commit();
+
+          // when
+          const results = await campaignProfilesCollectionParticipationSummaryRepository.findPaginatedByCampaignId(
+            campaign.id,
+          );
+
+          // then
+          expect(results.data[0]).to.deep.include({
+            pixScore: recentParticipationPixScore,
+            previousPixScore: null,
+            previousSharedAt: null,
+            evolution: null,
+          });
+
+          expect(results.data[1]).to.deep.include({
+            pixScore: otherOrganizationLearnerParticipation.pixScore,
+            previousPixScore: null,
+            previousSharedAt: null,
+            evolution: null,
+          });
+        });
+      });
+
+      describe('when participant has a third shared participation', function () {
+        it('should return latest participation and additionnal infos for participation before', async function () {
+          const oldParticipationPixScore = 20;
+          const oldParticipationSharedAt = new Date('2024-01-02');
+
+          databaseBuilder.factory.buildCampaignParticipation({
+            campaignId: campaign.id,
+            userId,
+            organizationLearnerId,
+            pixScore: oldParticipationPixScore,
+            isImproved: true,
+            createdAt: new Date('2024-01-01'),
+            sharedAt: oldParticipationSharedAt,
+            status: SHARED,
+          });
+          const inBetweenParticipationPixScore = 60;
```

Comment:
**suggestion** : renommer en `previous` ici. et les `old` dans les autres it sont en fait des `previous`